res/wrap.jl

```diff
@@ -0,0 +1,46 @@
+using Clang
+
+if length(ARGS) != 1
+  error("Pass the path to the Flux headers as the first argument")
+end
+
+header_dir = ARGS[1]
+isdir(header_dir) || error("$header_dir does not exist")
+
+const FLUX_INCLUDE = normpath(header_dir)
+
+# Fails with `include/flux/core/barrier.h:24:1: error: unknown type name 'flux_future_t'`
```

Comment:
Were you still running into this error?  If so, we can open a flux-core issue to get that fixed upstream.

Reply:
Yes running agains `0.23.0`:

```
vchuravy@odin ~/s/F/res (vc/tests)> julia --project=. wrap.jl /home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/barrier.h:24:1: error: unknown type name 'flux_future_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/connector.h:22:9: error: unknown type name 'flux_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/connector.h:26:44: error: unknown type name 'size_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/connector.h:28:38: error: unknown type name 'size_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/connector.h:31:43: error: unknown type name 'flux_msg_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/connector.h:32:5: error: unknown type name 'flux_msg_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/connector.h:40:1: error: unknown type name 'flux_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/connector.h:41:27: error: unknown type name 'flux_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/keepalive.h:18:1: error: unknown type name 'flux_msg_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/keepalive.h:20:34: error: unknown type name 'flux_msg_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_commit.h:33:1: error: unknown type name 'flux_future_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_commit.h:33:33: error: unknown type name 'flux_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_commit.h:34:33: error: unknown type name 'flux_kvs_txn_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_commit.h:36:1: error: unknown type name 'flux_future_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_commit.h:36:32: error: unknown type name 'flux_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_commit.h:38:32: error: unknown type name 'flux_kvs_txn_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_commit.h:41:34: error: unknown type name 'flux_future_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_commit.h:42:35: error: unknown type name 'flux_future_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_copy.h:30:1: error: unknown type name 'flux_future_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_copy.h:30:31: error: unknown type name 'flux_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_copy.h:47:1: error: unknown type name 'flux_future_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_copy.h:47:31: error: unknown type name 'flux_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_dir.h:38:36: error: unknown type name 'flux_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_dir.h:62:1: error: unknown type name 'bool'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_dir.h:66:1: error: unknown type name 'bool'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_dir.h:70:1: error: unknown type name 'bool'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_getroot.h:19:1: error: unknown type name 'flux_future_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_getroot.h:19:34: error: unknown type name 'flux_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_getroot.h:28:35: error: unknown type name 'flux_future_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_getroot.h:29:35: error: unknown type name 'flux_future_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_getroot.h:30:36: error: unknown type name 'flux_future_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_getroot.h:31:33: error: unknown type name 'flux_future_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_getroot.h:31:51: error: unknown type name 'uint32_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_lookup.h:18:1: error: unknown type name 'flux_future_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_lookup.h:18:33: error: unknown type name 'flux_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_lookup.h:20:1: error: unknown type name 'flux_future_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_lookup.h:20:35: error: unknown type name 'flux_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_lookup.h:23:26: error: unknown type name 'flux_future_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_lookup.h:24:33: error: unknown type name 'flux_future_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_lookup.h:25:30: error: unknown type name 'flux_future_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_lookup.h:26:34: error: unknown type name 'flux_future_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_lookup.h:27:30: error: unknown type name 'flux_future_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_lookup.h:27:54: error: unknown type name 'flux_kvsdir_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_lookup.h:28:34: error: unknown type name 'flux_future_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_lookup.h:32:38: error: unknown type name 'flux_future_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_lookup.h:39:29: error: unknown type name 'flux_future_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/service.h:34:1: error: unknown type name 'flux_future_t'
/home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/service.h:46:1: error: unknown type name 'flux_future_t'
[ Info: wrapping header: /home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core.h ...
[ Info: writing libflux_h.jl
[ Info: wrapping header: /home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/attr.h ...
[ Info: writing libflux_h.jl
[ Info: wrapping header: /home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/barrier.h ...
[ Info: writing libflux_h.jl
[ Info: wrapping header: /home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/buffer.h ...
[ Info: writing libflux_h.jl
[ Info: wrapping header: /home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/conf.h ...
┌ Warning: Skipping flux_conf_vunpack(const flux_conf_t *, flux_conf_error_t *, const char *, struct __va_list_tag *) due to unsupported argument: va_list
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:35
┌ Warning: Skipping VarArg Function CLCursor (CLFunctionDecl) flux_conf_unpack(const flux_conf_t *, flux_conf_error_t *, const char *, ...)
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:15
[ Info: writing libflux_h.jl
[ Info: wrapping header: /home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/connector.h ...
[ Info: writing libflux_h.jl
[ Info: wrapping header: /home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/content.h ...
┌ Warning: Skipping unnamed EnumDecl: CLCursor (CLEnumDecl) 
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:94
[ Info: writing libflux_h.jl
[ Info: wrapping header: /home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/event.h ...
┌ Warning: Skipping VarArg Function CLCursor (CLFunctionDecl) flux_event_unpack(const flux_msg_t *, const char **, const char *, ...)
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:15
┌ Warning: Skipping VarArg Function CLCursor (CLFunctionDecl) flux_event_pack(const char *, const char *, ...)
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:15
┌ Warning: Skipping VarArg Function CLCursor (CLFunctionDecl) flux_event_publish_pack(flux_t *, const char *, int, const char *, ...)
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:15
[ Info: writing libflux_h.jl
[ Info: wrapping header: /home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/flog.h ...
┌ Warning: Skipping flux_vlog(flux_t *, int, const char *, struct __va_list_tag *) due to unsupported argument: va_list
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:35
┌ Warning: Skipping VarArg Function CLCursor (CLFunctionDecl) flux_log(flux_t *, int, const char *, ...)
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:15
┌ Warning: Skipping flux_log_verror(flux_t *, const char *, struct __va_list_tag *) due to unsupported argument: va_list
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:35
┌ Warning: Skipping VarArg Function CLCursor (CLFunctionDecl) flux_log_error(flux_t *, const char *, ...)
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:15
[ Info: writing libflux_h.jl
[ Info: wrapping header: /home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/flux.h ...
[ Info: writing libflux_h.jl
[ Info: wrapping header: /home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/future.h ...
[ Info: writing libflux_h.jl
[ Info: wrapping header: /home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/handle.h ...
┌ Warning: Skipping unnamed EnumDecl: CLCursor (CLEnumDecl) 
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:94
┌ Warning: Skipping unnamed EnumDecl: CLCursor (CLEnumDecl) 
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:94
┌ Warning: Skipping unnamed EnumDecl: CLCursor (CLEnumDecl) 
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:94
[ Info: writing libflux_h.jl
[ Info: wrapping header: /home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/heartbeat.h ...
[ Info: writing libflux_h.jl
[ Info: wrapping header: /home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/job.h ...
┌ Warning: Skipping unnamed EnumDecl: CLCursor (CLEnumDecl) 
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:94
┌ Warning: Skipping unnamed EnumDecl: CLCursor (CLEnumDecl) 
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:94
[ Info: writing libflux_h.jl
[ Info: wrapping header: /home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/keepalive.h ...
[ Info: writing libflux_h.jl
[ Info: wrapping header: /home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs.h ...
[ Info: writing libflux_h.jl
[ Info: wrapping header: /home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_commit.h ...
[ Info: writing libflux_h.jl
[ Info: wrapping header: /home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_copy.h ...
[ Info: writing libflux_h.jl
[ Info: wrapping header: /home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_dir.h ...
[ Info: writing libflux_h.jl
[ Info: wrapping header: /home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_getroot.h ...
[ Info: writing libflux_h.jl
[ Info: wrapping header: /home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_lookup.h ...
┌ Warning: Skipping VarArg Function CLCursor (CLFunctionDecl) flux_kvs_lookup_get_unpack(int *, const char *, ...)
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:15
[ Info: writing libflux_h.jl
[ Info: wrapping header: /home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/kvs_txn.h ...
┌ Warning: Skipping flux_kvs_txn_vpack(flux_kvs_txn_t *, int, const char *, const char *, struct __va_list_tag *) due to unsupported argument: va_list
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:35
┌ Warning: Skipping VarArg Function CLCursor (CLFunctionDecl) flux_kvs_txn_pack(flux_kvs_txn_t *, int, const char *, const char *, ...)
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:15
[ Info: writing libflux_h.jl
[ Info: wrapping header: /home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/message.h ...
┌ Warning: Skipping unnamed EnumDecl: CLCursor (CLEnumDecl) 
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:94
┌ Warning: Skipping unnamed EnumDecl: CLCursor (CLEnumDecl) 
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:94
┌ Warning: Skipping unnamed EnumDecl: CLCursor (CLEnumDecl) 
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:94
┌ Warning: Skipping VarArg Function CLCursor (CLFunctionDecl) flux_match_asprintf(struct flux_match *, const char *, ...)
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:15
┌ Warning: Skipping VarArg Function CLCursor (CLFunctionDecl) flux_msg_pack(flux_msg_t *, const char *, ...)
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:15
┌ Warning: Skipping flux_msg_vpack(flux_msg_t *, const char *, struct __va_list_tag *) due to unsupported argument: va_list
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:35
┌ Warning: Skipping VarArg Function CLCursor (CLFunctionDecl) flux_msg_unpack(const flux_msg_t *, const char *, ...)
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:15
┌ Warning: Skipping flux_msg_vunpack(const flux_msg_t *, const char *, struct __va_list_tag *) due to unsupported argument: va_list
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:35
┌ Warning: Skipping unnamed EnumDecl: CLCursor (CLEnumDecl) 
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:94
┌ Warning: Skipping unnamed EnumDecl: CLCursor (CLEnumDecl) 
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:94
┌ Warning: Skipping unnamed EnumDecl: CLCursor (CLEnumDecl) 
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:94
[ Info: writing libflux_h.jl
[ Info: wrapping header: /home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/module.h ...
┌ Warning: Skipping unnamed EnumDecl: CLCursor (CLEnumDecl) 
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:94
[ Info: writing libflux_h.jl
[ Info: wrapping header: /home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/msg_handler.h ...
[ Info: writing libflux_h.jl
[ Info: wrapping header: /home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/panic.h ...
[ Info: writing libflux_h.jl
[ Info: wrapping header: /home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/plugin.h ...
┌ Warning: Skipping unnamed EnumDecl: CLCursor (CLEnumDecl) 
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:94
┌ Warning: Skipping VarArg Function CLCursor (CLFunctionDecl) flux_plugin_conf_unpack(flux_plugin_t *, const char *, ...)
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:15
┌ Warning: No Prototype for CLCursor (CLFunctionDecl) flux_plugin_arg_create() - assuming no arguments
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:13
┌ Warning: Skipping unnamed EnumDecl: CLCursor (CLEnumDecl) 
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:94
┌ Warning: Skipping VarArg Function CLCursor (CLFunctionDecl) flux_plugin_arg_pack(flux_plugin_arg_t *, int, const char *, ...)
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:15
┌ Warning: Skipping flux_plugin_arg_vpack(flux_plugin_arg_t *, int, const char *, struct __va_list_tag *) due to unsupported argument: va_list
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:35
┌ Warning: Skipping VarArg Function CLCursor (CLFunctionDecl) flux_plugin_arg_unpack(flux_plugin_arg_t *, int, const char *, ...)
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:15
┌ Warning: Skipping flux_plugin_arg_vunpack(flux_plugin_arg_t *, int, const char *, struct __va_list_tag *) due to unsupported argument: va_list
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:35
[ Info: writing libflux_h.jl
[ Info: wrapping header: /home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/reactor.h ...
┌ Warning: Skipping unnamed EnumDecl: CLCursor (CLEnumDecl) 
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:94
┌ Warning: Skipping unnamed EnumDecl: CLCursor (CLEnumDecl) 
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:94
┌ Warning: Skipping unnamed EnumDecl: CLCursor (CLEnumDecl) 
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:94
[ Info: writing libflux_h.jl
[ Info: wrapping header: /home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/request.h ...
┌ Warning: Skipping VarArg Function CLCursor (CLFunctionDecl) flux_request_unpack(const flux_msg_t *, const char **, const char *, ...)
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:15
[ Info: writing libflux_h.jl
[ Info: wrapping header: /home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/response.h ...
┌ Warning: Skipping VarArg Function CLCursor (CLFunctionDecl) flux_respond_pack(flux_t *, const flux_msg_t *, const char *, ...)
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:15
[ Info: writing libflux_h.jl
[ Info: wrapping header: /home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/rpc.h ...
┌ Warning: Skipping unnamed EnumDecl: CLCursor (CLEnumDecl) 
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:94
┌ Warning: Skipping VarArg Function CLCursor (CLFunctionDecl) flux_rpc_pack(flux_t *, const char *, uint32_t, int, const char *, ...)
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:15
┌ Warning: Skipping flux_rpc_vpack(flux_t *, const char *, uint32_t, int, const char *, struct __va_list_tag *) due to unsupported argument: va_list
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:35
┌ Warning: Skipping VarArg Function CLCursor (CLFunctionDecl) flux_rpc_get_unpack(flux_future_t *, const char *, ...)
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:15
[ Info: writing libflux_h.jl
[ Info: wrapping header: /home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/service.h ...
[ Info: writing libflux_h.jl
[ Info: wrapping header: /home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/subprocess.h ...
┌ Warning: Skipping unnamed EnumDecl: CLCursor (CLEnumDecl) 
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:94
┌ Warning: Skipping VarArg Function CLCursor (CLFunctionDecl) flux_cmd_argv_appendf(flux_cmd_t *, const char *, ...)
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:15
┌ Warning: Skipping VarArg Function CLCursor (CLFunctionDecl) flux_cmd_setenvf(flux_cmd_t *, int, const char *, const char *, ...)
└ @ Clang ~/.julia/packages/Clang/2Dqbm/src/wrap_c.jl:15
[ Info: writing libflux_h.jl
[ Info: wrapping header: /home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/types.h ...
[ Info: writing libflux_h.jl
[ Info: wrapping header: /home/vchuravy/src/spack/opt/spack/linux-archrolling-zen2/gcc-10.2.0/flux-core-0.23.0-6ujxj27lrueqy7xxaqmhdc5kxmr6m6xg/include/flux/core/version.h ...
[ Info: writing libflux_h.jl
```